### ISSUE TITLE
Animation

### ISSUE THREAD

**Issue by psenechal** [Cnilton/react-native-floating-label-input#10]
Hello...thanks for writing this library!

I noticed in the GIF example you have on the README, the labels do a nice little animated bounce when the input field is clicked on. In my implementation, I don't see that animation...the labels just instantly jump to their position. Is there some additional configuration I can add to duplicate the animation effect that you have in your GIF example?

Thanks =)

**Comment by Cnilton** [comment]
Hi @psenechal, there is no need for any additional configuration. Can you please inform the version of react-native and if possible a GIF of the problem?  It would help too if you could show a little bit of code to see if something is preventing the animation from happening. 

Thanks for contributing!

**Comment by psenechal** [comment]
Hello,
I'm using react-native: 0.63.2
The implementation is pretty simple:
```
                <FloatingLabelInput
                  label='Number of Bottles'
                  value={values.quantity}
                  keyboardType='number-pad'
                  onChangeText={handleChange('quantity')}
                />
```

The fields are wrapped in a Formik container.
I'll try to post a GIF of it...just need to figure out how to do that =)

Thanks!

**Comment by psenechal** [comment]
![floating_label](https://user-images.githubusercontent.com/1395106/94468135-d23f5000-0178-11eb-8afd-d3a305d9d0db.gif)

**Comment by sainyanhtay** [comment]
I'm facing animation issue too

**Comment by Cnilton** [comment]
Okay, it seems to be an issue with UIManager from NativeModules. I'll look into it

**Comment by sainyanhtay** [comment]
it's ok in android @Cnilton

**Comment by Cnilton** [comment]
Thanks, @sainyanhtay, I'll commit the changes briefly

**Comment by Cnilton** [comment]
Could you please test if it is still reproducible at version 1.2.4?

**Comment by psenechal** [comment]
Appears to be resolved in v1.2.4. Thanks for the update!

**Comment by Cnilton** [comment]
That's great, thanks for your contribution @psenechal, @sainyanhtay!
